hr_expense_sequence/data/hr_expense_data.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="seq_expense" model="ir.sequence">
+        <field name="name">Expense report sequence</field>
+        <field name="code">hr.expense.sheet</field>
+        <field name="padding" eval="4" />
+        <field name="prefix">EX</field>
```

Comment:
```suggestion
        <field name="prefix">EX</field>
        <field name="company_id" eval="False" />
```
I think it is better to set company_id to False from sequence so that it applies to all companies.

Reply:
@victoralmau I add it on this PR. Thank you for reviewed.